package.json

```diff
@@ -37,9 +37,10 @@
   "scripts": {
     "predeploy": "yarn build",
     "deploy": "gh-pages -d dist/",
-    "start": "parcel src/index.html",
+    "update:static": "cp src/wasm/multisig.wasm dist/multisig.wasm",
```

Comment:
should we use Parcel plugin so that it is updated automatically in dev mode?

https://www.npmjs.com/package/parcel-plugin-static-files-copy

Reply:
if you think it's necessary. I think many times plugins are overkill for copy one file like this?

Reply:
yeah, I guess given it's not going to change frequently should be fine as is